cmd/mcp-server/scenarios/README.md

```
@@ -0,0 +1,97 @@
+| independent-failures | Two unrelated component operators scaled to 0 | infrastructure/cluster-distress | 1099 | low |
+| missing-dependency | cert-manager operator scaled to 0 | unknown/unclassifiable | 3000 | low |
+| node-pressure | Worker node patched with MemoryPressure=True | infrastructure/node-pressure | 1005 | high |
+| operator-crash | ODH operator deployment scaled to 0 | infrastructure/operator | 1008 | high |
+| partial-failure | One component operator scaled to 0, others healthy | infrastructure/dsc-unhealthy | 1010 | medium |
+| resource-exhaustion | Impossibly low ResourceQuota applied to apps namespace | infrastructure/quota-oom | 1004 | high |
+
+## File Structure
+
+Each scenario consists of three files:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)